src/AuthEncoding/AuthEncoding.py

```diff
@@ -108,10 +109,7 @@ def generate_salt(self):
         # because of limitations of the binascii module.
         # 7 is what Netscape's example used and should be enough.
         # All 256 characters are available.
```

Comment:
Is this comment still valid?

Reply:
There's hardly any documentation for SSHA because it's seriously outdated and was mostly used by LDAP implementations. I don't feel confident to change that comment with something from a web search that may end up equally old or wrong.